### DISCUSSION TITLE
Tuxemon Comics: Chapter 5 released!!

### DISCUSSION THREAD

**Discussion by ultidonki** [Tuxemon/Tuxemon#2326] [Show and tell]
Preview of the first page:
![page_01_bubbles1](https://github.com/user-attachments/assets/2cc62c35-012e-4591-9ae8-ef7767c02b62)

Click here to see the 2nd page and the rest:
https://github.com/ultidonki/tuxemon_comics/blob/master/english/05_teddisun/page_02_bubbles.png

License and credits are on the last page.

All my comics are here:
https://github.com/ultidonki/tuxemon_comics

(I have no idea what happened just before the comic (in the Sampsack Forest), or what will happen after (with the new character who has a Bolt monster). I leave it open to other people to create their own stories/mods/whatever with those ideas. :smile: )

**Comment by Sanglorian** [comment]
Haha, fun! I like that it teaches the type interactions too.

**Comment by Davidliang11292** [comment]
Glad you updated it!

**Comment by Davidliang11292** [comment]
Glad you updated it!

**Comment by JaskRendix** [comment]
@ultidonki Hey, I'm not gonna lie, I had this crazy idea for a Tuxemon comic and I just can't get it out of my head. Maybe if I just describe what I see in my mind, it'll spark something in you? Like, I picture this: 

_A female trainer, black haired, tries to catch a wild [TRASTO](https://wiki.tuxemon.org/Trasto) with her [FLUMMBY](https://wiki.tuxemon.org/Flummby), despite her friend's (male) warning that TRASTO is an unsettling Tuxemon. However, TRASTO ends up eating FLUMMBY, and the smell of food attracts a massive swarm of 200 SAMPSACK. Black haired and her friends (all 3) barely manage to escape with FLUMMBY's Tuxe Ball, and they flee the area in fear of the wild SAMPSACK._

Black haired = Lily, sorry I forgot the name

**Page 1**

(Panel 1: A female trainer, LILY, with a FLUMMBY perched on her shoulder, stands with her friends, BLONDE HAIR GIRL and GUY, in a dense forest. They're all looking at a wild TRASTO in the distance.)

LILY: (excitedly) Oh wow, a TRASTO! I've always wanted to catch one!

GUY: (cautiously) Are you sure that's a good idea, Lily? TRASTO's are kinda...unsettling.

LILY: (dismissively) Ah, don't be such a worrier! FLUMMBY and I can handle it.

(Panel 2: Lily confidently approaches the TRASTO, FLUMMBY still on her shoulder. The TRASTO looks at them with an unblinking stare.)

**Page 2**

(Panel 1: Lily throws a Tuxe Ball, but TRASTO easily dodges it. FLUMMBY, sensing an opportunity, leaps off Lily's shoulder and into the air.)

LILY: (shouting) FLUMMBY, use your special move!

(Panel 2: FLUMMBY lands on TRASTO's head, but instead of attacking, TRASTO's transparent body starts to glow. FLUMMBY looks confused.)

**Page 3**

(Panel 1: TRASTO's body starts to envelop FLUMMBY, and the FLUMMBY lets out a terrified "Squeak!" as it's being digested.)

LILY: (horrified) NOOO! FLUMMBY!

GUY: (I-told-you-so) I said it was a bad idea...

(Panel 2: The smell of food wafts through the air, and suddenly, 200 SAMPSACK start pouring out of the forest, attracted to the scent.)

**Page 4**

(Panel 1: Lily and her friends are surrounded by the SAMPSACK, who are all chomping their jaws eagerly. Lily looks desperate.)

LILY: (frantically) We have to get out of here, NOW!

(Panel 2: Lily manages to retrieve FLUMMBY's Tuxe Ball, and they all make a run for it, with the SAMPSACK hot on their heels.)

**Page 5**

(Panel 1: The trio emerges from the forest, panting and scared. Lily looks down at FLUMMBY's Tuxe Ball, relieved.)

LILY: (exhaling) We made it...but next time, I'll listen to your advice.

GUY: (smirking) Anytime, Lily.

(Panel 2: The three friends share a nervous laugh as they walk away from the forest, while the SAMPSACK are seen in the background, still searching for food.)

**Comment by JaskRendix** [comment]
@ultidonki regarding the sequel, it can be this:

**Page 1**

(Panel 1: A rugged, grey-bearded, bald man, stands with his BOLT Tuxemon perched on his arm. He's dressed in a white lab coat, similar to the ones worn by scientists. He looks at the three trainers, Lily, a guy, and a blonde-haired girl, with a calculating gaze.)

Zesty: So, these are the Tuxemon trainers I keep hearing about... Verrry interesting...

(Panel 2: The trainer guy, looking concerned, takes a step forward.)

Guy: Who are you? You're not from around here, are you?

(Panel 3: The man's eyes glaze over, and a flashback sequence begins.)

(Panel 4: The man, dressed in a chef's hat and apron, is shown expertly chopping vegetables in a bustling kitchen.)

Man (thought bubble): Ah, the life of a cook... I was rrolling in dough, rrunning my own kitchen...

**Page 2**

(Panel 5: The man is shown expertly flipping pancakes, with a sizzling pan in the background.)

Man (thought bubble): I was known for my "Electric Eggs" – the fluffiest, most shocking breakfast in town... People would rrush to get a table...

(Panel 6: The man is shown proudly holding up a tray of perfectly baked croissants.)

Man (thought bubble): And my "Thunderbolt Tarts" were the talk of the town... R-rival chefs would rrage with envy...

(Panel 7: The flashback ends, and the man returns to the present, a sly grin spreading across his face.)

Man: I'm Zesty "Sparkplug" Slade, the most electrifying chef-turned-trainer you'll ever meet! Rready to rramp up the competition!

Lily: Uh, nice to meet you, Zesty?

Blonde-haired girl: (giggling) Sparkplug? That's so cute!

Guy: (rolling his eyes) Rright, because the world was missing a chef-turned-trainer with a Tuxemon that can electrocute us all...

Zesty: (chuckling) Ah, you're a clever one! Yeah, that's exactly what I mean. And just like a spark plug, I'm about to ignite the Tuxemon world!

Zesty: (smiling slyly) By the way, you, my friend, owe me a small fortune. You and your friends dined at my restaurant last week, and I never got paid. The bill comes out to... (dramatic pause) ...10,000$!

Guy: (stunned) What?! That's outrageous!

Zesty: (shrugging) Ah, but you have to pay for the best. And my "Electric Eggs" are worth every penny. (smirking) Now, are you going to pay up, or do we need to take this outside?

Blonde-haired girl: (eyes wide with shock) 10,000$?! That's... that's... (stunned silence)

Lily: (sarcastically) Oh, great, because we didn't have anything better to spend our money on. Like, I don't know, Tuxemon food or something.

(The scene ends with Zesty's confident smile, as the three trainers exchange worried glances.)

**Reply by ultidonki** [reply]
@JaskRendix 

Ah! Thank you very much for continuing the story, it makes me very happy!!! 

The black-haired girl and new guy didn't have names yet, so yes we can call them Lily and Zesty. :smile: 

The script and explanations are very detailed, so thank you for that - that helps a lot. 

If you had the time, I wonder if you could try drawing the panels, you don't need to have any artistic ability:
Just draw circles for faces, tuxemon and hands etc., as long as everything has a name I can draw it myself. 
(I can draw this myself if you don't want to, but it's actually easier if someone else wants to do it. Although I'll make changes and add stuff anyway, I can use someone else's work as a starting point) :smiley: 

![storyboard1](https://github.com/user-attachments/assets/6214d3e3-e00b-4856-bb7c-bfad42646809)

Anyone can suggest any stories they want to, by the way - however, I'll only turn them into comics if I have the time, and it takes me a long time so I won't make any promises to start or finish anything. But the more ideas we have, the more other artists/comic makers might be inspired to help out. :smile:
